apps/coordinator/ts/redis/__tests__/redis.service.test.ts

```diff
@@ -0,0 +1,122 @@
+import { ESupportedNetworks } from "../../common";
+import { RedisService } from "../redis.service";
+import { IScheduledPoll } from "../types";
+import { getPollKeyFromObject } from "../utils";
+
+const REDIS__GET_ALL_PREFIX = "*-test";
+
+const scheduledPoll: IScheduledPoll = {
+  maciAddress: "0xb83074Ac11fc569AC12F1b7D0C0a6809c3dc355b",
+  pollId: "5",
+  mode: 1,
+  chain: ESupportedNetworks.OPTIMISM_SEPOLIA,
+  endDate: 1752534000,
+};
+
+describe("RedisService", () => {
+  let service: RedisService;
+
+  beforeAll(async () => {
+    service = new RedisService();
+    await service.onModuleInit();
+    // Clean up before tests
+    const keys = await service.client.keys(REDIS__GET_ALL_PREFIX);
+    if (keys.length) {
+      await service.client.del(keys);
+    }
+  });
+
+  afterAll(async () => {
+    // Clean up after tests
+    const keys = await service.client.keys(REDIS__GET_ALL_PREFIX);
+    if (keys.length) {
+      await service.client.del(keys);
```

Comment:
Do we need the same logic to delete values in `beforeAll` and `afterAll`?

Reply:
Based on another comment, if we wanted isolated unit tests, we could consider `afterEach`